ComfortZoneApp/app/src/main/java/com/example/comfortzone/Utils/ParseUtil.java

```diff
@@ -31,8 +35,14 @@ public void done(List<LevelsTracker> objects, ParseException e) {
                     tracker.addEntry(newEntry);
                     tracker.increaseCount();
                     tracker.saveInBackground();
+                    updateTodayEntries(currentUser, newEntry);
                 }
             }
         });
     }
+
+    public static void updateTodayEntries (ParseUser currentUser, ComfortLevelEntry newEntry) {
```

Comment:
if it's not use elsewhere, use `private` - always be conservative and use the tightest accessibility